src/DotNet.Status.Web.Tests/Files/triage-items-with-diff-cat.body.txt

```diff
@@ -1,3 +1,21 @@
+Error message
+```
+We stopped hearing from agent Azure Pipelines 11. Verify the agent machine is running and has a healthy network connection. Anything that terminates an agent process, starves it for CPU, or blocks its network access can cause this error. For more information, see: https://go.microsoft.com/fwlink/?li
+```
+Builds effected
```

Comment:
Nit: Maybe "affected" would be a better word here given it's not "builds created because of the problem", but "builds where the problem was observed"?